src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+        EpochMilliseconds
+      </td>
+
+      <td>
+        The time the issue was activated: createdAt + gracePeriod.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        closedAt
+      </td>
+
+      <td>
+        EpochMilliseconds
+      </td>
+
+      <td>
+        The time the issue was closed.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        closedBy
+      </td>
+
+      <td>
+        String
+      </td>
+
+      <td>
+        The user that manually closed the issue.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        createdAt
+      </td>
+
+      <td>
+        EpochMilliseconds
+      </td>
+
+      <td>
+        The time the issue was created.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        description
+      </td>
+
+      <td>
+        [String!]!
+      </td>
+
+      <td>
+        A list of incident descriptions.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        entityGuids
+      </td>
+
+      <td>
+        [EntityGuid!]
+      </td>
+
+      <td>
+        A list of unique entity identifiers.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        entityNames
+      </td>
+
+      <td>
+        [String!]
+      </td>
+
+      <td>
+        A list of entity names.
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        entityTypes
+      </td>
+
+      <td>
+       [String!]
+      </td>
+
+      <td>
+        A list of entity types.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        environmentId
+      </td>
+
+      <td>
+        [Int!]!
+      </td>
+
+      <td>
+        See [our documentation](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/) to learn about environments.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        eventType
+      </td>
+
+      <td>
+        String!
+      </td>
+
+      <td>
+        An issue event that could be INCIDENT_ADDED, USER_ACTION, INCIDENT_CLOSED, etc...
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        incidentIds
+      </td>
+
+      <td>
+        [String!]
+      </td>
+
+      <td>
+        A list of incident IDs in this issue. In some cases there may be thousands of incidents in an issue.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        isCorrelated
+      </td>
+
+      <td>
+        Boolean!
+      </td>
+
+      <td>
+        True means that other issues were correlated to this one.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        isIdle
+      </td>
+
+      <td>
+        Boolean!
+      </td>
+
+      <td>
+        See our documentation to learn about idle issues.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        issueId
+      </td>
+
+      <td>
+        ID!
+      </td>
+
+      <td>
+        Issue identifier.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        origins
+      </td>
+
+      <td>
+        [String!]!
+      </td>
+
+      <td>
+        A list of origins of incidents like violations, anomalies, external etc...
```

Comment:
Just a heads up that we don't like to say 'etc.'. If there's a 'like', you don't need the 'etc.', can just be 'like violations, anomalies, and external'.